packages/transformers/html/src/dependencies.js

```diff
@@ -157,6 +149,62 @@ export default function collectDependencies(
       return node;
     }
 
+    if (tag === 'script' && attrs.src) {
+      let sourceType = attrs.type === 'module' ? 'module' : 'script';
+      let loc = node.location
+        ? {
+            filePath: asset.filePath,
+            start: node.location.start,
+            end: node.location.end,
+          }
+        : undefined;
+
+      let outputFormat = 'global';
+      if (attrs.type === 'module' && asset.env.shouldScopeHoist) {
+        outputFormat = 'esmodule';
+      } else {
+        delete attrs.type;
+      }
+
+      // If this is a <script type="module">, and not all of the browser targets support ESM natively,
+      // add a copy of the script tag with a nomodule attribute.
+      let copy: ?PostHTMLNode;
+      if (
+        outputFormat === 'esmodule' &&
+        !asset.env.supports('esmodules', true)
+      ) {
+        let attrs = Object.assign({}, node.attrs);
+        copy = {...node, attrs};
+        delete attrs.type;
+        attrs.nomodule = '';
+        attrs.src = asset.addURLDependency(attrs.src, {
+          // Keep in the same bundle group as the HTML.
+          priority: 'parallel',
+          env: {
+            sourceType,
+            outputFormat: 'global',
+            loc,
+          },
+        });
```

Comment:
Do these statements do anything? `attrs` is a shallow clone and it was already merged into `copy`

Reply:
`attrs` is referenced by `copy`, so this is modifying the attrs of the copy.